src/xpcc/architecture/platform/driver/core/avr/linkerscript.ld

```diff
@@ -0,0 +1,13 @@
+SECTIONS
+{
+	.xpcc_assertion : AT( ADDR(.text) + SIZEOF(.text) + SIZEOF(.data))
+	{
+		. = ALIGN(2);
+		__assertion_table_start = . + SIZEOF(.data);
```

Comment:
This is an issue of VMA vs. LMA. This is placed physically after data into Flash, but virtually after flash. ld is not really well documented, so maybe I just don't know any better. But the end result is the same anyways.